### PR TITLE
feat(SEO): exchange names in the graph are links

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue

We don't really tell search engines about related zones.

Fixes: AVO-557

## Description

This makes the zone name and flag in the bar breakdown graph a link so better content and crawler graphs can be created for SEO.

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
> Should we also make the flag here and the pill with the country name a link if it's a subzone?
> 
> image

@Alportan what do you think? Could be good but the pill is not shown on all subzones.

Either way I think that is best suited to a follow up PR to try and scope the changes to one thing at the time.

**Review by silkeholmebonnen — APPROVED** [review]
Nice work! 🌟 I just have a question and a comment but nothing blocking 🌞

File: web/src/features/charts/bar-breakdown/elements/Row.tsx [hunk 1]
```diff
@@ -10,74 +15,173 @@ import {
...
+  index,
+}: BaseProps) {
   // Don't render if the width is not positive
   if (width <= 0) {
     return null;
   }
-
   return (
-    <g className="row" transform={`translate(0, ${index * (ROW_HEIGHT + PADDING_Y)})`}>
+    <Group top={index * (ROW_HEIGHT + PADDING_Y)}>
```
[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply web/src/features/charts/bar-breakdown/elements/Row.tsx:100]: Group is just a wrapper around `<g>` but it has a much nicer API
```
top={}
```
vs
```
transform={`translate(0, ${})`}
```

File: web/src/features/charts/bar-breakdown/elements/Row.tsx [hunk 2]
```diff
@@ -10,74 +15,173 @@ import {
...
-      )}
-    </g>
+      {!Number.isFinite(value) && <FallbackQuestionMark scale={scale} />}
+    </Group>
+  );
+}
+
+type ProductionProps = { productionMode: ElectricityModeType } & BaseProps;
+
+export function ProductionRow({
```
[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply web/src/features/charts/bar-breakdown/elements/Row.tsx:120]: I'll update it 🙂

**Pull request merged**